lib/signet/oauth_2/client.rb

```diff
@@ -1196,6 +1201,14 @@ def normalize_timestamp(time)
           fail "Invalid time value #{time}"
         end
       end
+
+      def set_relative_expires_at(issued_at, expires_in)
+        self.expires_in = expires_in.to_i
```

Comment:
You can remove this line. Both `expires_in=` and `expires_at=` set the same state.